src/server/config/__tests__/schema.js

```diff
@@ -198,5 +212,83 @@ describe('Config schema', function () {
         expect(whitelist).to.contain('/path/to');
       });
     });
+
+    describe('cors', () => {
+      describe('production', () => {
+        const validateServerCors = value => validate(
+          value === undefined ? {} : { server: { cors: value } },
+          { dev: false }
+        );
+
+        it('defaults to false', () => {
+          const { error, value } = validateServerCors();
+          expect(error).to.be(null);
+          expect(value.server.cors).to.be(false);
+        });
```

Comment:
__nit:__ could you please insert a new line between tests: it's a bit easier to read and follows the style used in this file already?